google/cloud/bigquery/v2/minimal/internal/table_schema.h

```diff
@@ -67,8 +67,8 @@ struct TableFieldSchema {
   std::string description;
   std::string collation;
   std::string default_value_expression;
+  std::string max_length;
 
-  std::int64_t max_length = 0;
```

Comment:
While it is a `string` in JSON, it is in `int64` format:

![image](https://github.com/googleapis/google-cloud-cpp/assets/6241635/e4d130fd-b327-435a-84de-84a518555520)


https://developers.google.com/discovery/v1/type-format

The C++ representation should be `std::int64_t`.  The library should convert to and from a string when reading and storing this from JSON.  This is a common pattern, as JSON cannot represent integers larger than (about) $2^{52}$

Reply:
Thank you, I didn't know that.
I left types of `maxLength` (and also `precision` and `scale`) as `std::int64_t` and updated `to_json` and `from_json` functions.